scenes/lessons/7th_class/tarayin_artahaytutyun/tarayin_artayahtutyun.py

```diff
@@ -0,0 +1,305 @@
+        arrow_t.rotate(3/16 * PI).next_to(final_solution[-1], UR, buff=0.1)
+
+        time_zhamanak = VGroup(Tex('time = ', font_size=35), Tex('ժամանակ', font_size=35, tex_template=ARMTEX))
+        time_zhamanak.arrange(buff=0.1, aligned_edge=UP)
+        time_zhamanak.next_to(arrow_t, UR, buff=0.1).shift([0.1, -0.05, 0])
+
+        solution_with_v = Tex('700', ' - ', 'V', ' $\cdot$ ', '7')
+        solution_with_v.move_to([0, -2, 0]).align_to(final_solution, LEFT)
+
+        solution_v_60 = Tex('700', ' - ', '60', ' $\cdot$ ', '7', ' = ', '280')
+        solution_v_60.move_to([0, -2.75, 0]).align_to(solution_with_v, LEFT)
+        case_v_is_60 = VGroup(
+            Tex('V = 60'),
+            Tex('դեպքում', tex_template=ARMTEX)
+        )
+        case_v_is_60.arrange(buff=0.2, aligned_edge=UP).next_to(solution_v_60, LEFT, buff= 0.5)
+
+
+
+# ANIMATIONS  
+    
+    # գծագիր ու պայմաններ
+        self.play(
+            Create(road_endpoint_left),
+            Create(road),
+            Create(road_endpoint_right)
+        )
+        self.play(Write(brace_total))
+        self.play(Write(total_distance))
+        self.wait()
+
+        self.play(FadeIn(car))
+        self.add(car_trace, road_endpoint_left, car)
+        self.wait()
+
+        self.play(
+            AnimationGroup(
+                Create(speed_vector[0]),
+                Create(speed_vector[1:3]),
+                Write(speed_vector[3]),
+                lag_ratio=0.5, run_time=1.5
+            )
+        )
+        self.wait()
+
+        self.play(VGroup(car, speed_vector).animate.shift(12/7 * RIGHT), rate_func=linear)
+        self.play(Write(brace_remaining))
+        self.play(Write(harcakan))
+        self.wait()
+
+        self.add(paymanner, pahanj, dashed_line, line_hor, line_ver)
+        self.wait()
+
+        self.play(FadeOut(speed_vector))
+        self.wait()
+
+    # լուծում
+
+        # առաջին քայլ
+        self.play(Write(solution_first_step[0]))
+        self.wait(0.25)
+
+        self.play(Circumscribe(paymanner[1], fade_out=True))
+        self.play(Circumscribe(pahanj[1:], fade_out=True))
+        self.wait(0.25)
+
+        self.play(ReplacementTransform(paymanner[1][1:2].copy(), solution_first_step[1:2]))
+        self.play(Write(solution_first_step[2], run_time=0.5))
+        self.play(ReplacementTransform(pahanj[1:2].copy(), solution_first_step[3:4]))
+        self.play(Write(solution_first_step[4:]))
+        self.wait(0.5)
+
+        self.play(Write(brace_passed))
+        self.play(Write(passed_distance))
+        self.wait()
+
+        # երկրորդ քայլ
+        self.play(Write(solution_second_step[0]))
+        self.wait(0.25)
+
+        self.play(Wiggle(brace_remaining, run_time=1.25, rate_func=linear))
+        self.wait(0.5)
+        self.play(Wiggle(brace_total, run_time=1.25, rate_func=linear))
+        self.wait(0.5)
+        self.play(Wiggle(brace_passed, run_time=1.25, rate_func=linear))
+        self.wait(0.5)
```

Comment:
Same here, you can summarize this as

```python
for brace in [brace_remaining, brace_total, brace_passed]:
        self.play(Wiggle(brace, run_time=1.25, rate_func=linear))
        self.wait(0.5)
```

I think this is more readable, but of course, it's subjective

Reply:
սկզբում ռանթայմերը տարբեր էի դրել, հետո փոխեցի սաղ նույնը սարքեցի

Reply:
հաա, հետո կռնա էլի տարբեր դառնա՞։ Եթե չէ՝ սպես բաները ընդհանրացնելը հարմար է